apps/journeys-admin/src/components/ContainedIconButton/ContainedIconButton.stories.tsx

```diff
@@ -0,0 +1,61 @@
+import { Meta, Story } from '@storybook/react'
+import Paper from '@mui/material/Paper'
+import { ComponentProps } from 'react'
+import Stack from '@mui/material/Stack'
+import NoteAddIcon from '@mui/icons-material/NoteAdd'
+import AddIcon from '@mui/icons-material/Add'
+import EditIcon from '@mui/icons-material/Edit'
+import { simpleComponentConfig } from '../../libs/storybook'
+import { ContainedIconButton } from '.'
+
+const ContainedIconButtonDemo = {
+  ...simpleComponentConfig,
+  component: ContainedIconButton,
+  title: 'Journeys-Admin/ContainedIconButton'
+}
+
+const Template: Story<ComponentProps<typeof ContainedIconButton>> = ({
+  ...args
+}) => (
+  <Paper elevation={0} sx={{ p: 2 }}>
+    <Stack direction="row">
+      <ContainedIconButton {...args} />
+    </Stack>
+  </Paper>
+)
+
+export const Default = Template.bind({})
+Default.args = {
+  thumbnailIcon: <NoteAddIcon />,
+  actionIcon: <AddIcon />,
+  label: 'label',
+  description: undefined,
+  imageSrc: undefined,
+  imageAlt: undefined,
+  loading: false,
+  onClick: () => console.log('Clicked')
```

Comment:
we can use noop in this case since we don't need to worry about what the on click is doing. You can import it from the lodash library (its in our packages already)

```suggestion
  onClick: noop
```